docs/modules/security/pages/security-realms.adoc

```diff
@@ -296,6 +296,9 @@ Detailed logging for LDAP authentication can be enabled by
 configuring a more verbose logger level for the `com.hazelcast.security`
 package as described in the xref:security-debugging.adoc[Security Debugging section].
 
+NOTE: Hazelcast provided LDAP authentication implementation doesn't handle LDAP referals
+(i.e. references to other LDAP trees).
```

Comment:
Maybe better to put this way? :

"The LDAP authentication implementation provided by Hazelcast doesn't handle LDAP referrals, i.e., references to other LDAP trees.

Reply:
:+1: thx, will change